src/test/java/api/support/fixtures/PatronGroupExamples.java

```diff
@@ -2,24 +2,30 @@
 
 import api.support.builders.PatronGroupBuilder;
 
+import java.util.UUID;
+
 class PatronGroupExamples {
-  static PatronGroupBuilder regular() {
+ static PatronGroupBuilder regular() {
     return new PatronGroupBuilder("Regular Group", "Regular group");
   }
 
-  static PatronGroupBuilder alternative() {
+ static PatronGroupBuilder alternative() {
     return new PatronGroupBuilder("Alternative Group", "Alternative group");
   }
 
-  static PatronGroupBuilder staff() {
+ static PatronGroupBuilder staff() {
     return new PatronGroupBuilder("staff", "Staff users");
   }
 
-  static PatronGroupBuilder faculty() {
+ static PatronGroupBuilder faculty() {
     return new PatronGroupBuilder("faculty", "Faculty users");
   }
 
-  static PatronGroupBuilder undergrad(){
+ static PatronGroupBuilder undergrad() {
     return new PatronGroupBuilder("undergrad", "Undergraduate users");
   }
+
+ static PatronGroupBuilder staffWithId(UUID id) {
```

Comment:
Is this still needed? I didn’t notice any usages

Reply:
You are right, it`s unused.
I removed it